Editors/README.md

```diff
@@ -74,10 +74,13 @@ You will need the path to the `sourcekit-lsp` executable for the "command" secti
 
 There is an Emacs client for SourceKit-LSP in the [main Emacs LSP repository](https://github.com/emacs-lsp/lsp-sourcekit).
 
-## Vim 8
+## Vim 8 or Neovim
 
-Install [vim-lsp](https://github.com/prabirshrestha/vim-lsp). In your `.vimrc`, configure vim-lsp to use
-sourcekit-lsp for Swift source files like so:
+All methods below assume `sourcekit-lsp` is in your `PATH`. If it's not then replace `sourcekit-lsp` with the absolute path to the sourcekit-lsp executable.
+
+### vim-lsp
+
+In your `.vimrc`, configure vim-lsp to use sourcekit-lsp for Swift source files like so:
```

Comment:
The "Install vim-lsp" and link seems to have disappeared in this edit. Maybe these two sections should each contain a link to the underlying plugin - either vim-lsp or coc?

Reply:
It does look like I accidentally removed the link. Should be back now.

Reply:
Also added a link to coc.nvim :)